code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -895,10 +894,7 @@
 						/obj/item/attachable/reddot,
 						/obj/item/attachable/verticalgrip,
 						/obj/item/attachable/angledgrip,
-						/obj/item/attachable/gyro,
 						/obj/item/attachable/flashlight,
-						/obj/item/attachable/extended_barrel,
```

Comment:
The ZX is overpowered at the moment, there's no denying that, but this is outside of the scope of the PR and warrants its own, as it's not just de-escalating damage, but changing how the gun works.